styles/freedesktop/catppuccin.user.css

```diff
@@ -0,0 +1,107 @@
+/* ==UserStyle==
+@name freedesktop Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/freedesktop
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/freedesktop
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/freedesktop/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Afreedesktop
+@description Soothing pastel theme for www.freedesktop.org
+@author NK308
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('www.freedesktop.org') {
+  /* the 'www' subdomain is important, since we don't want to include the 'gitlab' subdomain  */
```

Comment:
```suggestion
```

Reply:
I wanted to avoid the problem of someone coming up with the idea to improve the theme by making it more general by *just* removing the subdomain. This is, why I made this comment, to announce the choice of the exact domain to be a conscious decision.

Reply:
I don't think that needs a comment. Domains are rarely changed.

Reply:
.

Reply:
This is not resolved...

Reply:
Must have been accidentally reverted when I solved [this](https://github.com/catppuccin/userstyles/pull/860#issuecomment-2092485806).